LICENSE

```diff
@@ -1,9 +1,11 @@
 ### Code license
 
-Copyright (c) 2013-2020, The Swedish Internet Foundation
-Copyright (c) 2013-2020, AFNIC
+Copyright (c) The Swedish Internet Foundation (<https://internetstiftelsen.se/en/>)
+Copyright (c) AFNIC (<https://www.afnic.fr/en/>)
```

Comment:
Why are the links marked up with both parentheses and angle brackets? Isn't it enough with just one of them?

Reply:
In this case I guess it is, but that is consistent with the LICENSE file in the other repositories.